libparsec/crates/platform_storage/Cargo.toml

```
@@ -39,6 +41,7 @@ serde-wasm-bindgen = { workspace = true }
 libparsec_tests_lite = { workspace = true }
 # Note `libparsec_tests_fixtures` enables our `test-with-testbed` feature
 libparsec_tests_fixtures = { workspace = true, default-features = false, features = ["test-with-platform-storage-testbed"] }
+libparsec_platform_storage = {path = ".",features = ["expose-test-methods"]}
```

Missing space around brace and after comma:

```suggestion
libparsec_platform_storage = { path = ".", features = ["expose-test-methods"] }
```